java-checks/src/main/java/org/sonar/java/checks/StandardCharsetsConstantsCheck.java

```diff
@@ -227,13 +234,35 @@ public void visitNode(Tree tree) {
     }
   }
 
+  List<JavaQuickFix> quickFixesOnMemberSelect(IdentifierTree identifierTree) {
+    Tree parent = identifierTree.parent();
+    if (parent.is(Tree.Kind.MEMBER_SELECT)) {
+      MemberSelectExpressionTree parentMemberSelect = (MemberSelectExpressionTree) parent;
+      return List.of(
+        JavaQuickFix.newQuickFix(REPLACE_WITH_STANDARD_CHARSETS + identifierTree.name() + "\"")
+          .addTextEdit(JavaTextEdit.replaceTree(parentMemberSelect.expression(), "java.nio.charset.StandardCharsets"))
```

Comment:
Using the [ImportSupplier](https://github.com/SonarSource/sonar-java/blob/4892839eea5c24a6ad45c4dcb8d7d9561377680e/java-checks/src/main/java/org/sonar/java/checks/helpers/QuickFixHelper.java#L246), we should be able to add insert `StandardCharsets` with a minimal amount of text.
See this [check](https://github.com/SonarSource/sonar-java/blob/4892839eea5c24a6ad45c4dcb8d7d9561377680e/java-checks/src/main/java/org/sonar/java/checks/CollectionImplementationReferencedCheck.java#L157) for an example.